Description: not calling git as part of the build process
Author: Sylvestre Ledru <sylvestre@debian.org>
Forwarded: not-needed
Last-Update: 2023-04-17

--- a/make/build-common.xml
+++ b/make/build-common.xml
@@ -81,19 +81,7 @@
         <property name="jogl.build.number" value="manual"/>
         <property name="jogl.build.id"     value="${version.timestamp}"/>
         <mkdir dir="${tempdir}" />
-        <exec dir="." executable="git" logError="true" failonerror="false" failifexecutionfails="false"
-                  output="${tempdir}/localbranch.raw">
-            <arg line="branch --no-color"/>
-        </exec>
-        <exec dir="." executable="sed" logError="true" failonerror="false" failifexecutionfails="false"
-                  outputproperty="jogl.build.branch">
-            <arg line="-e '/^[^*]/d' -e 's/* \(.*\)/\1/' ${tempdir}/localbranch.raw"/>
-        </exec>
         <property name="jogl.build.branch" value="manual"/> <!-- fallback -->
-        <exec dir="${project.root}" executable="git" logError="true" failonerror="false" failifexecutionfails="false"
-                  outputproperty="jogl.build.commit">
-            <arg line="rev-parse HEAD"/>
-        </exec>
         <property name="jogl.build.commit" value="manual"/> <!-- fallback -->
 
         <property name="jogl.version"         value="${jogamp.version.base}-b${jogl.build.number}-${version.timestamp}" />
